### PR TITLE
Document failover for validator update

### PR DESCRIPTION
**Pull request by clay-aptos**
### Description

Addressing issue #5287 

### Test Plan

Local build and test

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/5555)

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   * [Aptos CLI](../../../cli-tools/aptos-cli-tool/install-aptos-cli.md)
+1. Copy the configuration files between the two nodes. See the files in the [validator setup](running-validator-node/index.md) documentation you used for the full list.
+
+## Failover
+
+Just before the next epoch change:
+1. Update DNS to [swap the node network addresses on-chain](staking-pool-operations.md#3-update-validator-network-addresses-on-chain).
+1. Turn down the validator node and validator full node.
+1. Restart the former validator full node with the validator node configuration.
+1. Observe that even before DNS changes take effect that outbound connections participate in consensus and proposals.
```
Comment (davidiw) [inline developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:32]: I think what we want to say here is that only outbound connections will form

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. Update your validator fullnode with the latest version of the:
+   * [required packages Aptos depends upon](../../../guides/getting-started#prepare-development-environment)
+   * [Aptos CLI](../../../cli-tools/aptos-cli-tool/install-aptos-cli.md)
+1. Copy the configuration files between the two nodes. See the files in the [validator setup](running-validator-node/index.md) documentation you used for the full list.
+
+## Failover
+
+Just before the next epoch change:
+1. Update DNS to [swap the node network addresses on-chain](staking-pool-operations.md#3-update-validator-network-addresses-on-chain).
+1. Turn down the validator node and validator full node.
```
Comment (davidiw) [inline developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:30]: in my mind, we likely have one or more VFNs. perhaps we should explicitly label the new VFN as the VFN intended to replace the validator.

**Review by davidiw — APPROVED** [review]
overall seems fine, but I want a PE @perryjrandall or @sherry-x to review this.

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Update Aptos Validator Node"
+slug: "update-validator-node"
+---
+
+# Update Aptos Validator Node via Failover
```
Comment (movekevin) [inline developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:6]: Should we also mention that unless the code is modified, running backup nodes doesn't have slashing risks? This is different from running a validator node for some other networks.

**Review by movekevin — APPROVED** [review]

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md (hunk 3 above)
Comment (JoshLind) [reply developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:6]: I don't think we should mention this. I suspect we'll actually want to do some form of slashing/rejections if someone runs a backup validator in the future :/

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:32]: Done. Is it clear ending at form?

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:30]: Done. I think. :-)
Please take a look.

**Comment by clay-aptos** [comment]
Asking for @perryjrandall or @sherry-x 's approval per David.

**Pull request merged**